commons/src/main/java/fr/free/nrw/commons/upload/UploadController.java

```diff
@@ -140,7 +140,8 @@ protected Contribution doInBackground(Void... voids /* stare into you */) {
             @Override
             protected void onPostExecute(Contribution contribution) {
                 super.onPostExecute(contribution);
-                uploadService.queue(UploadService.ACTION_UPLOAD_FILE, contribution);
+                //temporarily disabled for debugging
```

Comment:
Yeah, I was waiting for all of the other changes to be approved, changing this last so I can test all the other changes first. :) Is the rest of the pull request okay?